### ISSUE TITLE
Favicon is not showing

### ISSUE THREAD

**Issue by maski89** [Webstrates/Webstrates#56]
The favicon set in a webstrate doesn't show after page reload. It falls back to the Webstrates favicon (green "W").

## Example:

**Webstrate**: https://webstrates.cs.au.dk/spicy-mole-31/

When removing the `<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">` from the `<head>` and inserting it again, the favicon will show. Upon reload it will revert back to the Webstrates favicon.

**Comment by kbadk** [comment]
Do you know if this is a regression and if so, when did it happen? I'm unable to reproduce the issue (i.e. favicons work as expected) locally, but I can reproduce it on webstrates.cs.au.dk, but not on demo.webstrates.net and hikaru.cs.au.dk , either.

**Comment by maski89** [comment]
I don't know when exactly it happened, but around November last year it was still working (I remember the favicon worked then in Cards & Boards).

For me it doesn't matter, the favicon isn't showing on neither https://webstrates.cs.au.dk/Codestrate/ , https://hikaru.cs.au.dk/Codestrate/ nor https://demo.webstrates.net/favicon-test/ . Also tested it on iOS Chrome 64, Windows Chrome 64 and Android Chrome 65. The specified [favicon ](https://demo.webstrates.net/favicon-test/favicon.ico)isn't showing, just the green "W".

---

As of writing this I recognized some strange behavior: When I right-click on the links above and select "Open in new tab" the favicon will show in the tab opening in the background. However, if I then switch to the tab and reload it, the favicon will disappear again.
So I'm really not sure what the issue is. Maybe it's also something Chrome related. I don't know.

**Comment by kbadk** [comment]
I've went back to commits from September and the behaviour seems to be the same. Also, Chrome doesn't seem to be consistent with what it does. If I navigate to a simple webstrate with just a favicon (not a Codestrate), the custom favicon will be shown about 50% of the times.

When you overwrite the existing favicon, Chrome also doesn't update it, even if you've disabled caching. In fact, even if you navigate to the new favicon in the URL to force an update, the tab doesn't update the favicon until you create a new tab. Just reloading the tab does nothing.

Safari seems to ignore favicons injected after load at all . However, if you load a page that has a favicon in it, and then remove the favicon from the DOM, then Safari will happily remove the favicon from the tab as well. So there's no hope of getting favicons working in Safari it seems since everything is injected into the page after load.

Then there's Opera. Opera will also ignore the favicon when it's initially injected into the DOM, but if it's later removed and re-added, then Opera will update the tab bar to reflect the new DOM icon. This must also be some sort of timing issue as we're re-adding it in the exact same way as it was initially added, so why doing the same thing again a little later works makes no sense.

Finally we have Firefox. Firefox does it almost perfectly. It updates the favicon when the page loads, it also accepts reinjecting another favicon into the DOM later on. When removing the favicon from the DOM, it doesn't remove it from the tab bar, though, but that is also kind of an edge case.

In summary, we now only support Firefox. Jokes aside, I don't think this will be an issue I'll be able to solve within a reasonable amount of time, especially not when none of the 4 browsers can agree on how to handle injected favicons in the first place. 

It didn't break as a result of Webstrates changing, but rather changes to Chrome. I'm closing the issue, but crossing my fingers that Chrome (and the other browsers eventually will come to a sensible agreement). Sorry.
